src/views/AddTenant/addTenant.js

```diff
@@ -158,16 +168,31 @@ export const AddTenant = () => {
             occupants: "",
           }}
           validationSchema={validationSchema}
+          validate={validateForm}
           validateOnBlur={false}
-          onSubmit={(values, { setSubmitting }) => {
+          onSubmit={(values, { setSubmitting, resetForm }) => {
             const toSubmit = {
               ...values,
-              dateTimeStart,
-              dateTimeEnd,
+              occupants: values.occupants || null,
+              unitNum: values.unitNum || null,
+              propertyID: propertySelection[0].key,
```

Comment:
When you try to submit a new tenant w/no lease through the UI, this line will cause the site to hang if a property is not selected.  The console error I received was a Formik TypeError, "propertySelection[0] is undefined"

I was able to get rid of this error and successfully create an "unhoused" tenant by changing the line to:
```suggestion
propertyID: propertySelection[0] ? propertySelection[0].key : null,
```

I'll leave it to your discretion whether this is the best way to handle this, but we definitely should not have to select a property to create a tenant.

Reply:
I agree. Property should be considered as part of the lease form. Tenants should not have a direct relationship with property, they're only assigned a property through a lease.